docs/preprocessing.md

```diff
@@ -349,12 +349,9 @@ input(on:input="{bar}")
 
 ### scss, sass
 
-The `scss/sass` preprocessor accepts the default sass options alongside two other props:
+The `scss/sass` preprocessor accepts the options listed in the [Sass `LegacyStringOptions` API reference](https://sass-lang.com/documentation/js-api/interfaces/LegacyStringOptions) with some exceptions: the `file` and `data` properties are not supported. Instead, use the `prependData` property if you want to prepend some content to your `scss` content.
```

Comment:
Wasn't sure if

> ... with some exceptions: the `file and `data` properties are not supported. Instead, use the `prependData` property if you want to prepend some content to your `scss` content.

should perhaps be changed to something more like

> 
> ... with some exceptions. The following options are not supported, as `svelte-preprocess` overrides them:
> 
> * `includePaths`
> * `sourceMap`
> * `sourceMapEmbed`
> * `omitSourceMapUrl`
> * `outFile`
> * `outputStyle`
> * `file`
> * `data`
> * `importer`
> 
> If you want to prepend some content to your `scss`, you can use `prependData` option that's specific to `svelte-preprocess`.

Just because it looks like all these options get clobbered: https://github.com/sveltejs/svelte-preprocess/blob/4218419a145c29248d682cddc3eda557409ec38b/src/transformers/scss.ts#L46-L62

I'm not really familiar with this codebase though, so perhaps I've misunderstood, so I'll leave it up to the maintainers if they want to add this level of additional detail.

Reply:
I don't think omitting these is a problem. as most don't make sense in the context of a preprocessor for the style content.

Explaning each of them:
`includePaths` can be passed (as you can see from the `options.includePaths` on line 46. Most of the other props are not that relevant given the context sass is being used. We need to force source maps, so those should be set regardless. IIRC, the `outFile` must be set due to a limitation of how sass returns the imported dependencies. If there's no `outFile`, the dependency list comes empty. `outputStyle` will not minify the output, which I think should be expected in this case.